.github/workflows/capgen_unit_tests.yaml

```diff
@@ -0,0 +1,22 @@
+name: Capgen Unit Tests
```

Comment:
@gold2718 @climbfuji @mwaxmonsky @peverwhee 
It would be nice if we had a CI test in the feature_capgen branch that ran the ccpp_prebuild.

We do run ccpp_prebuild as part of the ccpp-scm and ccpp-physics CI (e.g. https://github.com/ufs-community/ccpp-physics/blob/ufs/dev/.github/workflows/ci_scm_ccpp_prebuild.yml). One could easily modify this to run ccpp_prebuild using the feature_capgen fork, which would let us know if capgen development breaks the existing prebuild.
Thoughts? I can add this test to this PR if there is interest. Personally, I'd like to see this.

Reply:
I personally think this is a great idea!  Our original thought was that if we could "unify" the testing system first then it would make unifying the actual framework easier.  Of course @peverwhee and @mwaxmonsky are the ones actually responsible for all of this so I'm happy to let them make the final decision (at least for the NCAR side).

Reply:
Note that we already have `ccpp_prebuild.py` tests in `main`:
1. In `stub` to build a `CCPP` stub that exercises the framework. See `README.md` in this directory. Annoyingly, this seems to be broken right now, at least on my macOS (I'll look into fixing this later today).
2. In subdirectory `tests` (not the `s` at the end, the `test` directory is for `capgen`). These can be run via
```
cd tests
PYTHONPATH="$PWD/../scripts:$PWD/../scripts/parse_tools:$PYTHONPATH" python3 test_mkstatic.py
PYTHONPATH="$PWD/../scripts:$PWD/../scripts/parse_tools:$PYTHONPATH" python3 test_metadata_parser.py
```

Reply:
I have a local workflow file that can do this similar to ccpp-physics but after discussing with @peverwhee, we are going to add that in a separate PR because it fails in the current state but successfully runs when merged with the main branch.

Reply:
created issue #500 for prebuild CI integration